src/app/aside/aside.component.scss

```diff
@@ -1,14 +1,27 @@
 @import "src/assets/styles/variables.scss";
 
 aside{
-    background: honeydew;
+    background-color: $tundora-color;
     height: calc(100vh - #{$header-height} - #{$footer-height});
     overflow-y: auto;
 }
 
-.active{
-    background-color: bisque;
+.sidebar{
+    padding: 0;
+    
 }
-ul{
-    margin: 0;
+
+.nav{
+    li{
+        a{
```

Comment:
@Yuriy-27  you can write it shorter like instead three levels if some levels are empty
.nav li a {
...
}